tracer/src/Datadog.Trace/Debugger/Snapshots/DebuggerSnapshotSerializer.cs

```diff
@@ -215,8 +213,6 @@ private static void WriteFieldsInternal(object source, JsonWriter jsonWriter, Ca
         {
             // According the the debugger snapshot json structure RFC, the "this" value, unlike all other complex types,
```

Comment:
Is this comment still relevant?

Reply:
It's not, I'll delete it in the following PR I'm currently working on. Thanks! Resolving for now.